src/OpenLoco/Windows/PromptBrowseWindow.cpp

```diff
@@ -926,6 +927,12 @@ namespace OpenLoco::Ui::PromptBrowse
                     return;
             }
 
+            // Save last directory
+            fs::path lastDirectory = fs::u8path(&_directory[0]);
+            Config::getNew().last_save_path = lastDirectory.u8string();
+            Config::writeNewConfig();
+            Environment::resolvePaths();
+
```

Comment:
Perhaps this could just be moved to the end of this function so it applies to both the load and save success paths?